e2e-tests/IIoTPlatform-E2E-Tests/deploy/IoTHubPublisherDeployment.cs

```diff
@@ -0,0 +1,205 @@
+        public async Task CreateOrUpdateLayeredDeploymentAsync() {
+            await CreateOrUpdateConfigurationAsync(new ConfigurationModel {
+                Id = "__default-opcpublisher-standalone",
+                Content = new ConfigurationContentModel {
+                    ModulesContent = CreateLayeredDeployment(true, true)
+                },
+                SchemaVersion = kDefaultSchemaVersion,
+                TargetCondition = TargetConditionStandalone +
+                    " AND tags.os = 'Linux'",
+                Priority = 1
+            }, true, new CancellationToken());
+        }
+
+        /// <summary>
+        /// Get base edge configuration
+        /// </summary>
+        /// <returns></returns>
+        private IDictionary<string, IDictionary<string, object>> CreateLayeredDeployment(bool isLinux, bool isStandalone) {
+            var registryCredentials = "";
+            if (!string.IsNullOrEmpty(_context.ContainerRegistryConfig.DockerServer) &&
+                _context.ContainerRegistryConfig.DockerServer != "mcr.microsoft.com") {
+                var registryId = _context.ContainerRegistryConfig.DockerServer.Split('.')[0];
+                registryCredentials = @"
+                    ""properties.desired.runtime.settings.registryCredentials." + registryId + @""": {
+                        ""address"": """ + _context.ContainerRegistryConfig.DockerServer + @""",
+                        ""password"": """ + _context.ContainerRegistryConfig.DockerPassword + @""",
+                        ""username"": """ + _context.ContainerRegistryConfig.DockerUser + @"""
+                    },
+                ";
+            }
+
+            // Configure create options per os specified
+            string createOptions;
+            if (isLinux) {
+                if (isStandalone) {
+                    createOptions = JsonConvert.SerializeObject(new {
+                        Hostname = "publisher_standalone",
+                        Cmd = new[] {
+                        "PkiRootPath=/mount/pki",
+                        "--aa",
+                        "--pf=/mount/published_nodes.json"
+                    },
+                        HostConfig = new {
+                            Binds = new[] {
+                            "/mount:/mount"
+                            }
+                        }
+                    }).Replace("\"", "\\\"");
+                }
+                else {
+                    createOptions = JsonConvert.SerializeObject(new {
+                        Hostname = "publisher",
+                        Cmd = new[] {
+                        "PkiRootPath=/mount/pki",
+                        "--aa"
+                    },
+                        HostConfig = new {
+                            Binds = new[] {
+                            "/mount:/mount"
+                            }
+                        }
+                    }).Replace("\"", "\\\"");
+                }
```

Comment:
Can be removed, ne need to create layered deployment for orchestrated mode

Reply:
fixed